shell/platform/linux/fl_renderer.cc

```diff
@@ -10,6 +10,8 @@
 G_DEFINE_QUARK(fl_renderer_error_quark, fl_renderer_error)
 
 typedef struct {
+  FlView* view;
```

Comment:
This has created an unnecessary circular dependency. fl_renderer_get_view is only used at initialization, so the view can just be passed through those methods.

Reply:
Since It's used in a few places in construction in X11 and in map/unmap functions on Wayland (which you can now see in my draft PR: https://github.com/flutter/engine/pull/20629/files), I thought it was cleaner to allow accessing it from the renderer. We *could* pass it around everywhere and store a copy in `FlRendererWayland` instead of `FlRenderer`, but I don't see the advantage of that.

Reply:
I'm reviewing https://github.com/flutter/engine/pull/20629 and it looks like a lot of the `fl_renderer_get_view` calls aren't necessary.